[2025-10-02T23:56:14.411+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T23:56:14.458+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:55:00+00:00 [queued]>
[2025-10-02T23:56:14.474+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:55:00+00:00 [queued]>
[2025-10-02T23:56:14.475+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-02T23:56:14.504+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_numss> on 2025-10-02 23:55:00+00:00
[2025-10-02T23:56:14.525+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=688) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T23:56:14.530+0000] {standard_task_runner.py:63} INFO - Started process 690 to run task
[2025-10-02T23:56:14.529+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_taskk', 'task_numss', 'scheduled__2025-10-02T23:55:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpkyuy36xa']
[2025-10-02T23:56:14.537+0000] {standard_task_runner.py:91} INFO - Job 182: Subtask task_numss
[2025-10-02T23:56:14.569+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-02T23:56:14.702+0000] {task_command.py:426} INFO - Running <TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:55:00+00:00 [running]> on host 452c0657f430
[2025-10-02T23:56:15.046+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_taskk' AIRFLOW_CTX_TASK_ID='task_numss' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T23:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-02T23:55:00+00:00'
[2025-10-02T23:56:15.053+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T23:56:15.058+0000] {logging_mixin.py:188} INFO - random num is saved
[2025-10-02T23:56:15.060+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T23:56:15.062+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T23:56:15.091+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_taskk, task_id=task_numss, run_id=scheduled__2025-10-02T23:55:00+00:00, execution_date=20251002T235500, start_date=20251002T235614, end_date=20251002T235615
[2025-10-02T23:56:15.189+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T23:56:15.242+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T23:56:15.247+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
